app/storage/encrypted_questionnaire_storage.py

```diff
@@ -1,14 +1,13 @@
 import hashlib
```

Comment:
Have removed the storage factory and now instantiate the encrypted storage directly. Also removed the environment variable which toggles encrypted storage since it's no longer used.